tapchannel/aux_funding_controller.go

```diff
@@ -1058,6 +1058,18 @@ func (f *FundingController) completeChannelFunding(ctx context.Context,
 
 	log.Debugf("Funding PSBT pre funding: %s", spew.Sdump(fundingPsbt))
 
+	// Before we proceed, we'll make sure the fee rate we're using is above
+	// the min relay fee.
+	minRelayFee, err := f.cfg.ChainWallet.MinRelayFee(ctx)
```

Comment:
I think we should move this check into `processFundingReq`, directly after the `!supportsAssetChans` check.
Otherwise we kick off a bunch of temporary state and p2p communication, just to find out something we already knew earlier.